### PR TITLE
Add API for ProjectErrorList in session.ts

### PR DESCRIPTION
**Pull request by zhengbli**
Related to #3567, merge to master instead of release-1.5.
Incorporated CR feedbacks.

### CODE REVIEW

File: src/server/session.ts [hunk 1]
```diff
@@ -919,6 +964,13 @@ namespace ts.server {
                         responseRequired = false;
                         break;
                     }
+                    case CommandNames.GeterrForProject: {
```
Comment (danquirk) [inline src/server/session.ts]: GetErrForProject?

Comment (zhengbli) [inline src/server/session.ts]: There is a function called ``Geterr`` already, just to be consistent with that.

**Comment by zhengbli** [comment]
@mhegazy could you take a look?

**Comment by mhegazy** [comment]
:+1:  sorry for the delay.

**Pull request merged**